### ISSUE TITLE
Rendering of water use wrong color

### ISSUE THREAD

**Issue by livioavalle** [enzet/map-machine#120]
![test](https://user-images.githubusercontent.com/12595115/158208371-98947da6-c194-4d0e-a7fb-628b760153c1.svg)
Hi, I am trying to render an area near a port. But it came out with the water grey. Is this a bug or I missed something in the command used?
I used this command:
`map-machine render -b=8.9153,44.3872,8.9490,44.4011 -z=16 -o=out/test.svg`

**Comment by enzet** [comment]
Hello? @livioavalle! Sorry for the very late reply.

Yeah, I know this issue. This is definitely a bug. Coastline is quite complicate feature and I still have no stable algorithm for rendering it, so I decided not to render it at all. I will return to it as soon as possible.

**Comment by habi** [comment]
Somehow related to https://github.com/enzet/map-machine/issues/143

**Comment by enzet** [comment]
You're right.  I'll close #143 as a duplicate and will continue with this issue.

**Comment by enzet** [comment]
No, heuristics don't work in too many cases. Now Map Machine tries to request data through Overpass API to complete water relations.

> Since I discovered that resolving water polygons is unsolvable in the general case when we have incomplete water relations, I decided to add an additional request via the Overpass API to complete these relations. If Overpass isn't available, we will still try to draw water rectangles properly using heuristics.
>
> Overpass can be disabled by `--no-overpass` option.
